.ci/.lintr.R

```diff
@@ -73,7 +73,7 @@ rm(dt_linters)
 # TODO(lintr#2172): Glob with lintr itself.
 exclusions = c(local({
   exclusion_for_dir <- function(dir, exclusions) {
-    files = file.path("..", list.files(dir, pattern = "\\.(R|Rmd|Rraw)$", full.names=TRUE, recursive=TRUE))
```

Comment:
why revert this :)

Reply:
Oh, I had to re-fork the repository and I did not notice this when I did my commit.
I'll do a separate pull request for the main package .po files, since the discussion here is mainly about vignettes.